### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by OverSx**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @JuliaRoUTC22

### CODE REVIEW

**Comment by d-makarenko** [comment]
Прошу исправить замечания

File: .github/Console_homework_2/Console_homework_2/Program.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            Console.SetCursorPosition(centerX, 0);
+            Console.WriteLine(Title);
+
+            Console.WriteLine("Введите параметр R.");
+            int r;
+            r = int.Parse(Console.ReadLine());
+
+            for (double x = -5; x <= 9; x += 0.3) //шаг 0.2
+            {
+                if (x < -5)
```
[the remark this answers is not in the document]
Comment (JuliaRoUTC22) [reply .github/Console_homework_2/Console_homework_2/Program.cs]: Опечатка...

[the remark this answers is not in the document]
Comment (JuliaRoUTC22) [reply .github/Console_homework_2/Console_homework_2/Program.cs]: Исправленная версия загружается?

[the remark this answers is not in the document]
Comment (d-makarenko) [reply .github/Console_homework_2/Console_homework_2/Program.cs]: Нет, последний коммит - 7 дней назад

[the remark this answers is not in the document]
Comment (d-makarenko) [reply .github/Console_homework_2/Console_homework_2/Program.cs]: А, понятно, было исправлено 7 дней назад, но исправленную версию загрузила только сейчас?

[the remark this answers is not in the document]
Comment (JuliaRoUTC22) [reply .github/Console_homework_2/Console_homework_2/Program.cs]: Нет, я, как исправила неделю назад, сразу и загрузила.

[the remark this answers is not in the document]
Comment (JuliaRoUTC22) [reply .github/Console_homework_2/Console_homework_2/Program.cs]: image

[the remark this answers is not in the document]
Comment (JuliaRoUTC22) [reply .github/Console_homework_2/Console_homework_2/Program.cs]: Новая версия тогда не загрузилась?

**Comment by SergeyTeslov** [comment]
Оценка: 5/5

**Pull request merged**